sdk/monitor/monitor-opentelemetry/src/index.ts

```diff
@@ -73,10 +73,11 @@ export function useAzureMonitor(options?: AzureMonitorOpenTelemetryOptions) {
 }
 
 /**
- * Shutdown Azure Monitor Distro
+ * Shutdown Azure Monitor Open Telemetry Distro
+ * @see https://github.com/open-telemetry/opentelemetry-js/blob/0229434cb5a3179f63c021105f36270ae7897929/experimental/packages/opentelemetry-sdk-node/src/sdk.ts#L398
  */
-export function shutdownAzureMonitor() {
-  sdk?.shutdown();
+export async function shutdownAzureMonitor(): Promise<void> {
+  await sdk?.shutdown();
```

Comment:
Like @JacksonWeber mentioned, we should not wait here, this code was fire and forget before and functionality should remain, you can return sdk?.shutdown() here so the Promise could be waited or not, ensuring browserSdkLoader is disposed first.

Reply:
Just pushed an update, let me know if this is better @hectorhdzg @JacksonWeber